modin/core/execution/ray/generic/io/io.py

```diff
@@ -118,9 +118,23 @@ def to_csv(cls, qc, **kwargs):
         if not cls._to_csv_check_support(kwargs):
             return BaseIO.to_csv(qc, **kwargs)
 
-        # The partition id will be added to the queue, for which the moment
-        # of writing to the file has come
-        queue = Queue(maxsize=1)
+        @ray.remote
+        class SignalActor:  # pragma: no cover
```

Comment:
I think we should extract it to `core/execution/ray/common/utils.py` as it's not specific to `to_csv()`.